pkg/reloader/reloader.go

```diff
@@ -154,6 +158,18 @@ func New(logger log.Logger, reg prometheus.Registerer, o *Options) *Reloader {
 				Help: "Total number of reload requests that failed.",
 			},
 		),
+		lastReloadSuccess: promauto.With(reg).NewGauge(
+			prometheus.GaugeOpts{
+				Name: "reloader_last_reload_successful",
```

Comment:
Why querying counter of all reloads does not give us that? 🤔

Reply:
@bwplotka - I don't understand this sorry. We maintain a counter for total reloads and failed reloads. How would I determine from those if the latest retry was a success?

Reply:
Just looking at the mixin for bad config alert for Prometheus it achieves it using similar https://github.com/prometheus/prometheus/blob/main/documentation/prometheus-mixin/alerts.libsonnet#L8

Still unsure how we would guard against false alarm by determining without determining if they very last attempt was a success.

For example say I have a 90% failure rate on reloads, I might alert on that, although I wouldn't care if I knew the last reload attempt actually forced the reload. How would we achieve the same guard with the timestamp?

Reply:
Right - we can do that with ` increase(total[5m])  - increase(fails[5m])  > 0` no? 

But happy to stick to standard if we are familiar with it